src/domUpdates.js

```diff
@@ -71,10 +72,22 @@ const displayTotal = (currentTraveler) => {
   userTotal.innerHTML = `Total Spent on trips: $${currentTraveler.totalSpent}`
 }
 const displayNewTripForm = () => {
-  showHide([formWrapper, yourAccountButton], [newTripButton, dashboard])
+  showHide(
+    [formWrapper, yourAccountButton],
+    [newTripButton, loginDisplay, dashboard]
+  )
 }
 const displayAccount = () => {
-  showHide([dashboard, newTripButton], [formWrapper, yourAccountButton])
+  showHide(
```

Comment:
Your show/hide functions are great; awesome use of query selectors here within the same function.